csrc/scheduler/normalization.cpp

```diff
@@ -685,10 +685,9 @@ std::shared_ptr<ReductionParams> innerPersistentHeuristic(
   // blocks and buffers
   if (vectorize && blocksPerKernel > device_multiprocessor_count &&
       batches_per_block_inner_reduction > 1) {
-    constexpr int64_t reg_allocation_granularity = 256;
```

Comment:
Curious why this value was removed.

Reply:
That granularity is implicitly used in utility function `getRegPerThreadGivenThreadsPerSM` and `getThreadsPerSMGivenRegPerThread`

Reply:
This reminds me to recalculate register per thread after adjusting registers by 85%. Added the following part:
```
    // recalculate register_count_minimum using blocks_per_sm_maximum
    // this may increase register_count_minimum due to allocation granularity
    // e.g. 61 -> 64
    register_count_minimum = getRegPerThreadGivenThreadsPerSM(
        blocks_per_sm_maximum * threads_per_block);
```